### ISSUE TITLE
Slight display issue in the curve card in v1.34, of a graph with 0% in curve.

### ISSUE THREAD

**Issue by NormanPCN** [Rem0o/FanControl.Releases#1221]
Screenshot of the card.
https://i.postimg.cc/k5kvp9NQ/screenshot-287.jpg

Screenshot of the curve.
https://i.postimg.cc/GtjPtyWK/screenshot-288.jpg

The graph in the card is displayed half out at the bottom, 0% area. Previously the bottom 0% was fully displayed above the bottom of the card.

**Relevant hardware specs and setup**
Gigabyte Aorus z390 master, 9900k, Nvidia RTX 3070.

**Comment by Rem0o** [comment]
Yeah that's one consequence of the new look I gave the card that I'm not 100% happy with. I think I should add a little offset to the whole curve just to elevate the 0% line a few pixels over, while keep that "flush" look with the bottom of the card.

**Comment by Rem0o** [comment]
![image](https://user-images.githubusercontent.com/14118956/198905784-0c8e27ae-586e-4354-8664-f4697bde5202.png)

I think this will work : added a bit of padding at the bottom of the graph and tuned the gradiant. 

Still get the fade to nothing/flush look, while giving visibility to the 0% line. 

In the previous picture, the bottom section of the curve is 0%.
